### PR TITLE
fix(cesium-integration-build): ol map switched to cesium, build no ci flag

### PR DESCRIPTION
**Pull request by alebinson**

### CODE REVIEW

File: src/discrete-layer/components/map-container/map-container.tsx [hunk 1]
```diff
@@ -1,11 +1,39 @@
 import React, { useState } from 'react';
-import { Polygon } from 'geojson';
-import { DrawType } from '@map-colonies/react-components';
+import { Feature, FeatureCollection, Point, Polygon } from 'geojson';
+import { find } from 'lodash';
+import * as turf from '@turf/helpers';
```
Comment (asafMasa) [inline src/discrete-layer/components/map-container/map-container.tsx:4]: why import *?
it is bad for tree shaking

File: src/discrete-layer/components/map-container/map-container.tsx [hunk 2]
```diff
@@ -16,30 +44,100 @@ export interface MapContainerProps {
...
-    props.handlePolygonSelected(polygon);
+  const createDrawPrimitive = (type: DrawType): IDrawingObject => {
+    return {
+      type: type,
+      handler: (drawing: IDrawingEvent): void => {
+        const timeStamp = getTimeStamp();
+
+        setIsDrawing(false);
+
+        // console.log('primitive-->',drawing.primitive, 'geometry-->', (drawing.geojson as Feature).geometry);
```
Comment (asafMasa) [inline src/discrete-layer/components/map-container/map-container.tsx:69]: remove remark

**Review by asafMasa — CHANGES_REQUESTED** [review]
👍 good job

File: package.json [hunk 3]
```diff
@@ -33,7 +33,11 @@
     "confd": "node ./confd/generate-config.js",
     "confd:prod": "node ./confd/generate-config.js --environment production",
     "confd:prod:indocker": "node ./confd/generate-config.js --environment production --indocker",
-    "copyconf": "copyfiles ./confd/* ./build"
+    "copyconf": "copyfiles ./confd/* ./build",
+    "copyassets:all": "yarn run copyassets:cesium && yarn run copyassets:md-fonst && yarn run copyassets:components",
```
[the remark this answers is not in the document]
Comment (alebinson) [reply package.json:37]: As we talk it should be seamless, assets from **installed node_modules** are copied to *piblic* dir

File: .eslintignore [hunk 4]
```diff
@@ -0,0 +1 @@
+public/*
```
[the remark this answers is not in the document]
Comment (alebinson) [reply .eslintignore:1]: Done

File: src/discrete-layer/components/map-container/map-container.tsx (hunk 1 above)
Comment (alebinson) [reply src/discrete-layer/components/map-container/map-container.tsx:4]: Done

File: src/discrete-layer/components/map-container/map-container.tsx (hunk 2 above)
Comment (alebinson) [reply src/discrete-layer/components/map-container/map-container.tsx]: Done

File: src/discrete-layer/components/layers-results/cell-renderer/layer-details.cell-renderer.tsx [hunk 5]
```diff
@@ -17,8 +17,9 @@ interface DetailsProp {
 
 const stringRenderer: FormatterFunc = (val): string => val !== undefined ? val.toString() : '';
 const dateRenderer: FormatterFunc = (date): string => {
-  return date !== undefined
-    ? moment(new Date(date.toLocaleString())).format('DD/MM/YYYY HH:mm')
+  // eslint-disable-next-line
+  return (date !== undefined && ("toISOString" in (date as any)))
+    ? moment(date).format('DD/MM/YYYY HH:mm')
```
Comment (asafMasa) [inline src/discrete-layer/components/layers-results/cell-renderer/layer-details.cell-renderer.tsx:22]: use same time format in all app (config)

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: src/discrete-layer/components/layers-results/cell-renderer/layer-details.cell-renderer.tsx (hunk 5 above)
Comment (alebinson) [reply src/discrete-layer/components/layers-results/cell-renderer/layer-details.cell-renderer.tsx:22]: Done

**Review by asafMasa — APPROVED** [review]

**Pull request merged**